packages/E2ETest/wdio/pages/HomePage.ts

```diff
@@ -5,23 +5,35 @@
 
 import { BasePage, By } from './BasePage';
 import {
-  SEARCH_BOX,
-  BACK_BUTTON,
+  APIS_NAV_BUTTON,
+  COMPONENTS_NAV_BUTTON,
+  TESTER_LIST_SEARCH_BOX,
 } from '@react-native-windows/tester/js/examples-win/LegacyTests/Consts';
 
 class HomePage extends BasePage {
-  goToTestPage(page: string) {
+  goToComponentExample(example: string) {
+    this.waitForElementLoaded(COMPONENTS_NAV_BUTTON);
+    By(COMPONENTS_NAV_BUTTON).click();
+    this.goToExampleOnCurrentTab(example);
+  }
+
+  goToApiExample(example: string) {
+    this.waitForElementLoaded(APIS_NAV_BUTTON);
+    By(APIS_NAV_BUTTON).click();
+    this.goToExampleOnCurrentTab(example);
+  }
+
+  private goToExampleOnCurrentTab(example: string) {
     // Filter the list down to the one test, to improve the stability of selectors
-    this.waitForElementLoaded(SEARCH_BOX);
-    let editBox = By(SEARCH_BOX);
-    editBox.setValue(page);
-    let pageItem = By(page);
+    this.waitForElementLoaded(TESTER_LIST_SEARCH_BOX);
+    const editBox = By(TESTER_LIST_SEARCH_BOX);
+    editBox.setValue(example);
+    const pageItem = By(example);
     pageItem.click();
-    super.waitForElementLoaded(BACK_BUTTON);
-  }
 
-  backToHomePage() {
-    this.homeButton.click();
+    // Make sure we've launched the example by waiting until the search box is
+    // no longer present
+    this.waitForElementHidden(TESTER_LIST_SEARCH_BOX);
```

Comment:
>this.waitForElementHidden(TESTER_LIST_SEARCH_BOX); [](start = 4, length = 50)

I worry that checking for the lack of an element could lead to false positives, or fragility down the road.  Can we consider adding some element to each page like a Title which always has the same testID?  Then we could wait for that element to exist, which I think would be a better confirmation the page is loaded.  I think RN guys would accept that sort of change if we upstreamed it.  I won't block on this, feel free to track this as a follow-up issue if necessary.